packaging/archlinux/PKGBUILD

```diff
@@ -1,5 +1,30 @@
 pkgname=librdkafka
-pkgver=master
+pkgver=1.0.0.RC5.r11.g3cf68480
 pkgrel=1
-pkgdesc=The Apache Kafka C/C++ client library
-arch=('i686' 'x86_64')
+pkgdesc='The Apache Kafka C/C++ client library'
+url='https://github.com/edenhill/librdkafka'
+license=('BSD')
+arch=('x86_64')
+source=('git+https://github.com/edenhill/librdkafka#branch=master')
+sha256sums=('SKIP')
+depends=(glibc libsasl lz4 openssl zlib zstd)
+makedepends=(bash git python)
+
+pkgver() {
+	cd "$pkgname"
+	git describe --long --tags --match "v[0-9]*" | sed 's/^v//;s/\([^-]*-g\)/r\1/;s/-/./g'
```

Comment:
Might want to anchor and require digit for the first regex: `"^v[0-9]"`

Reply:
It's not a regex, but a glob:
```
$ git describe --long --tags --match "^v[0-9]*"
fatal: No names found, cannot describe anything.
$ git describe --long --tags --match "v[0-9]*"
v1.0.0-RC5-12-g3f5d5d8e
```
From `man git-describe`
```
       --match <pattern>
           Only consider tags matching the given glob(7) pattern, excluding the "refs/tags/" prefix. If used with --all, it
           also considers local branches and remote-tracking references matching the pattern, excluding respectively
           "refs/heads/" and "refs/remotes/" prefix; references of other types are never considered. If given multiple times,
           a list of patterns will be accumulated, and tags matching any of the patterns will be considered. Use --no-match to
           clear and reset the list of patterns.
```